.github/workflows/deploy-dev.yml

```diff
@@ -53,7 +54,7 @@ jobs:
           RESOURCE: .nais/nais-dev.yml
           VAR: image=${{ env.docker_image }}
       - name: post-deploy
-        if: github.ref == 'refs/heads/main'
+        #if: github.ref == 'refs/heads/main'
```

Comment:
Hvorfor kommenterte du ut denne? Post-deploy påvirker ikke dev-deploys fra feature branch. Den forbereder bare noen ting for generering av Release'en i neste steg. Så når du kommenterer ut denne så gjør du bare feature-branch deploys marginalt tregere.

Reply:
Egentlig ingen god forklaring på det. Gikk nok litt fort i svingene, men takk for oppklaringen!